src/tests/ie_test_utils/common_test_utils/CMakeLists.txt

```diff
@@ -62,6 +72,28 @@ function(add_common_utils ADD_TARGET_NAME)
     target_include_directories(${ADD_TARGET_NAME} SYSTEM PUBLIC ${IE_TESTS_ROOT}/ie_test_utils)
 
     target_compile_definitions(${ADD_TARGET_NAME} PUBLIC ${ARGN})
+
+    if(ENABLE_CONFORMANCE_PGQL)
+#Uncomment to be able use statically linked libpq
```

Comment:
Why do we need this comment and code? I don't believe that somebody will change it.

Reply:
removed